src/viur/core/modules/user.py

```diff
@@ -163,6 +161,12 @@ class UserSkel(skeleton.Skeleton):
         readOnly=True,
         defaultValue=0,
     )
+    # Authenticator OTP Apps (like Authy)
+    otp_secret_key = CredentialBone(
```

Comment:
The name `otp_secret_key` clashes with `otp_secret`. Can this be renamed to something different first, like `otp_app_secret`?